packages/vite-plugin/src/index.ts

```diff
@@ -16,6 +16,8 @@ import {
 const styleUpdateEvent = (fileId: string) =>
   `vanilla-extract-style-update:${fileId}`;
 
+const virtualPrefix = 'vanilla-extract:';
```

Comment:
Based on the plugin conventions you linked to, should this prefix be `virtual:vanilla-extract/`?

Reply:
You're right, it looks like that it should be `virtual:vanilla-extract/` + path. (I first understood it as `virtual:` + path, and virtual could be replaced with the namespace, e.g. `vanilla-extract`). Anyway, it doesn't work with `virtual:vanilla-extract/` due to the `/` in the end. The relative path is being resolved incorrectly, e.g.

```
virtual:vanilla-extract/../design-system/src/theme.css.ts$$$@***/components.vanilla.js
```

is being resolved to 

```
/@id/design-system/src/theme.css.ts$$$@***/components.vanilla.js
```

which is resolved to

```
design-system/src/theme.css.ts$$$@***/components.vanilla.js
```

Which doesn't exist.

Removing `/` from `virtual:vanilla-extract/` (or replacing it with `:` or `-`) solves the problem. Maybe we should stick to something like `/@ve-css` (as described in https://github.com/seek-oss/vanilla-extract/pull/457) or use something like `/@vanilla-extract` ?

Thanks heaps for your help and love your work 🥇

Reply:
Could we make the prefix `virtual:vanilla-extract:` then?

Reply:
@mattcompiles set to `virtual:vanilla-extract:` thank you